haiku-apps/zookeeper/zookeeper-1.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Execute commands on multiple files"
+DESCRIPTION="ZooKeeper is a glue application and a Tracker add-on that lets \
+you specify a shell command or script to be executed on a set of files."
+HOMEPAGE="https://github.com/HaikuArchives/ZooKeeper/"
+COPYRIGHT="2004,2005 Jonas Sundstrom"
+LICENSE="Public Domain"
+REVISION="1"
+srcGitRev="0b8fa5e0b36c4a52502b46f58a0f4f452139407c"
+SOURCE_URI="https://github.com/HaikuArchives/ZooKeeper/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="99c3085c636e2eddeb1a4b9a598707fc0f7b5fcbb56f57c472c5c5cc922b47f6"
+SOURCE_FILENAME="ZooKeeper-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="ZooKeeper-$srcGitRev"
```

Comment:
What's the current version of Zookeeper?  There was no commits for 5 years, and then you updated the makefile, rdef, etc.  In the rdef is the version info as it was last released.  There's been no functional changes since then, so that would be the version.  You should open an issue on the Zookeeper repo to create a tagged release of that version x.x.x. Then once someone tags a release for you, you can update the source like here to point to a released version, rather than today's master.  I think we've tried tagging via pull request but it didn't go through.  If you need to change any code, such as in an about box to also show current version you could also do that first.

Reply:
@scottmc Got the version!
Making a new issue on ZooKeeper repo right now.

Reply:
@bach5000 Actually the about box says "2.1" not "2.1.0" :)